README.md

```diff
@@ -19,7 +19,14 @@ trace by default since it seems to confuse too many users).
 
 Installing
 ----------
-
+Preparation: Install git(if not installed); Create working directory to save the source; Pull the respository
+ 
+    $ apt-get install git
+    $ mkdir workingcopy/
+    $ cd workingcopy/ 
+    $ git init .
+    $ git pull https://github.com/tmbdev/ocropy
```

Comment:
That seems too complicated and `git clone` should be enough for everything here.